link/src/main/java/com/stripe/android/link/ui/inline/LinkInlineSignup.kt

```diff
@@ -139,79 +146,135 @@ internal fun LinkInlineSignup(
-                        modifier = Modifier.padding(end = 8.dp),
-                        enabled = enabled
-                    )
-                    Column {
-                        Text(
-                            text = stringResource(id = R.string.stripe_inline_sign_up_header),
-                            style = MaterialTheme.typography.body1.copy(fontWeight = FontWeight.Bold),
-                            color = MaterialTheme.colors.onSurface
-                                .copy(alpha = contentAlpha)
-                        )
-                        Text(
-                            text = stringResource(R.string.stripe_sign_up_message, merchantName),
-                            modifier = Modifier
-                                .fillMaxWidth()
-                                .padding(top = 4.dp),
-                            style = MaterialTheme.typography.body1,
-                            color = MaterialTheme.colors.onSurface
-                                .copy(alpha = contentAlpha)
-                        )
-                    }
-                }
+                        .fillMaxWidth()
+                        .padding(top = 4.dp),
+                    style = MaterialTheme.typography.body1,
+                    color = MaterialTheme.colors.onSurface
+                        .copy(alpha = contentAlpha)
+                )
             }
+        }
+    }
+}
 
-            AnimatedVisibility(visible = expanded) {
-                Column(
-                    modifier = Modifier.padding(
-                        start = 16.dp,
-                        end = 16.dp,
-                        bottom = 16.dp,
-                    )
-                ) {
-                    LinkInlineSignupFields(
-                        sectionError = sectionError?.errorMessage,
-                        emailController = emailController,
-                        phoneNumberController = phoneNumberController,
-                        nameController = nameController,
-                        signUpState = signUpState,
-                        enabled = enabled,
-                        isShowingPhoneFirst = false,
-                        emailFocusRequester = emailFocusRequester,
-                        requiresNameCollection = requiresNameCollection,
-                        errorMessage = errorMessage,
-                        didShowAllFields = didShowAllFields,
-                        onShowingAllFields = { didShowAllFields = true },
-                    )
-
-                    AnimatedVisibility(visible = signUpState == InputtingRemainingFields) {
-                        LinkTerms(
-                            isOptional = false,
-                            isShowingPhoneFirst = false,
-                            modifier = Modifier.padding(top = 16.dp),
-                            textAlign = TextAlign.Start,
-                        )
-                    }
-                }
+@Composable
```

Comment:
Can you add/update (screenshot) tests for this?

Reply:
This was done to move code around in order to comply with `detekt` rules so the screenshot test results didn't change. I could add individual tests for these components?

Reply:
Less worried about testing the specific components as much as testing the new functionality. 

It's scary to see a behavior change be the intent of a PR without new/updated tests.